.ci/jobs/defaults.yml

```diff
@@ -0,0 +1,19 @@
+---
+
+##### GLOBAL METADATA
+
+- meta:
+    cluster: apm-ci
+
+##### JOB DEFAULTS
+
+- job:
+    view: APM-CI
```

Comment:
Where is this view in the Jenkins UI?

Reply:
it is not created, I think it was lost in the migration to JJBB, we could add it to the library repo